node/src/index.ts

```diff
@@ -166,13 +166,10 @@ export class LocalConnection implements Connection {
    * @param embeddings An embedding function to use on this Table
    */
   async openTable<T> (name: string, embeddings: EmbeddingFunction<T>): Promise<Table<T>>
-  async openTable<T> (name: string, embeddings?: EmbeddingFunction<T>): Promise<Table<T>> {
+  async openTable<T> (name: string, embeddings?: EmbeddingFunction<T>, awsCredentials?: AwsCredentials): Promise<Table<T>>
```

Comment:
is this a union type kind of parameter?
 can we do `credentialProvider?:  AwsCredentials | GcpCredentislInTheFuture`?
 
 Just feel that this interface is not very future proof.